data/channels.csv

```diff
@@ -20545,6 +20557,7 @@ SABCEducation.za,SABC Education,,SABC,South African Broadcasting Corporation,ZA,
 SABCNews.za,SABC News,,,,ZA,,,c/ZA,eng,news,FALSE,2013-01-01,,,http://www.sabc.co.za/,https://i.imgur.com/liLta8j.png
 SABCSport.za,SABC Sport,,SABC,South African Broadcasting Corporation,ZA,,,c/ZA,eng,sports,FALSE,,,,https://www.sabcsport.co.za/,https://upload.wikimedia.org/wikipedia/commons/3/38/SABC%2BSport%2B300x169.jpg
 SaberMas.us,Saber Mas,,,,US,,,c/US,spa,,FALSE,,,,https://watch.plex.tv/live-tv/channel/saber-mas,https://i.imgur.com/EfyaOXz.png
+SabhaTV.in,Sabha TV,,,,IN,,,c/IN,mal,legislative,FALSE,2020-08-17,,,https://sabhatv.com/,https://i.ibb.co/4NcM4hV/512px-Sabha-TV-Logo-svg.png

```

Comment:
```suggestion
SabhaTV.in,Sabha TV,,,,IN,,,c/IN,mal,legislative,FALSE,2020-08-17,,,https://sabhatv.com/,https://i.imgur.com/iaegMMm.png
```
Removed the transparent background around the logo.

Reply:
The original image already has a transparent background!